frontend/src/packages/orders/orders-api.ts

```diff
@@ -50,7 +50,7 @@ class OrdersApi extends HttpApi {
         method: 'POST',
         contentType: ContentType.JSON,
         payload: JSON.stringify(payload),
-        hasAuth: false,
+        hasAuth: true,
       },
```

Comment:
not registered user will not be able create order?

Reply:
This parameter adds only a header to the request. And if not, then how else will the order know that a registered user wants to place it?

Reply:
Of course, I tested it before the merge.